src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -405,13 +410,14 @@ static void create_stub_global_symbol(
   new_symbol.base_name = symbol_basename;
   new_symbol.type = symbol_type;
   new_symbol.type.set(ID_C_class, class_id);
+  new_symbol.type.set(ID_C_access, ID_public); // Guessed.
```

Comment:
What does guessed mean here? Please add some comment of why it's public rather than default for example.